src/components/SubMenu.vue

```diff
@@ -1,17 +1,25 @@
 <template>
-  <q-menu auto-close>
+  <q-menu ref="qMenu" :anchor="anchor" :self="self">
```

Comment:
You may want to position the main menu below the toolbar and the submenus next to the main menu. To achieve this, we need to propagate the anchor and self.